spec/contribute_spec.rb

```diff
@@ -1,11 +1,15 @@
 require 'spec_helper'
 
 describe Contribute do
+  it 'runs some test' do
+    expect(true).to eq(true)
+  end
+
   it 'has a version number' do
     expect(Contribute::VERSION).not_to be nil
   end
 
-  it 'does something useful' do
-    expect(false).to eq(true)
+  it 'should return some non-empty value for stars for a valid search query' do
+    expect(Contribute::Finder.new('language:ruby').find).not_to be_empty
```

Comment:
Yeah that's what I thought too. I thought of using stubs but stub wasn't the right thing here. Ok I will try it out and update it.